tests/integration/models/reddit/test_live.py

```diff
@@ -156,3 +156,14 @@ def test_close(self, _):
         with self.recorder.use_cassette(
                 'TestLiveThreadContribution_close'):
             thread.contrib.close()
+
+
+class TestLiveUpdateContribution(IntegrationTest):
+    @mock.patch('time.sleep', return_value=None)
```

Comment:
The access_token request doesn't set rate limit headers, so it does not result in a sleep. Typically only tests with more than one API interaction involve a sleep. I always just try it without, and it'll raise an exception if a sleep falls through without being patched.